crates/pcb-kicad/src/lib.rs

```diff
@@ -38,19 +48,28 @@ mod paths {
             })
             .replace(
                 "~",
-                dirs::home_dir()
-                    .unwrap_or_default()
-                    .to_str()
-                    .unwrap_or_default(),
+                path_helpers::home_dir().as_str(),
             )
     }
 }
 
 #[cfg(target_os = "windows")]
 mod paths {
+    use std::path::Path;
+    use crate::path_helpers;
+
     pub(crate) fn python_interpreter() -> String {
-        std::env::var("KICAD_PYTHON_INTERPRETER")
-            .unwrap_or_else(|_| r"C:\Program Files\KiCad\9.0\bin\python.exe".to_string())
+        let paths = vec![
+            std::env::var("KICAD_PYTHON_INTERPRETER")
+                .unwrap_or_else(|_| "python.exe".to_string()),
```

Comment:
Would this default to using the system `python.exe` if `KICAD_PYTHON_INTERPRETER` isn't defined?

Reply:
As it stands, it will only work in an `virtualenv` directory layout, but a fallback to scan the path (equivalent of`which`) would facilitate what you are suggesting